src/utils/geo.js

```diff
@@ -200,11 +200,33 @@ export function isProperGeo(countryList = [], name = undefined) {
 	);
 }
 
-export default {
+/**
+ * Select Instant Globals to send them to DFP (aka Ad Manager).
+ *
+ * @param {string[] | undefined} wfKeyVals mapping of Instant Globals (to send) to value in DFP
+ */
+export function getDfpLabradorKeyvals(wfKeyVals) {
+	if (!wfKeyVals || !wfKeyVals.length) {
```

Comment:
What do you think about moving it to context? Like `context.get('options.geo.labradorKeyVals')`

Reply:
Moving what exactly?

Reply:
The way key-vals list is passed to this function. So on client side instead of:
```
const labrador = adProductsUtils.getDfpLabradorKeyvals(instantGlobals.get('wgAdDriverLABradorDfpKeyvals'));
```
do something like:
```
context.set('options.geo.labradorKeyVals', instantGlobals.get('wgAdDriverLABradorDfpKeyvals'))
```
somewhere in the ad-context setup. And then:
```
const labrador = adProductsUtils.getDfpLabradorKeyvals();
```

also @wikia/ad-products doesn't need to be aware of ant "WF". I may be just key-vals list

Reply:
In my opinion the more the function depends on it's inputs the better. Your solution requires setting state (`context.options.geo.labradorKeyVals`) prior to testing.

Good point about "WF" 👁

Reply:
I agree, but for me it looks like app configuration ("list of labrador variables to use as key-vals")